packages/docs/vite.config.ts

```diff
@@ -28,23 +28,23 @@ export default defineConfig({
   },
 })
 
-function copyPiniaPlugin(): Plugin {
-  return {
-    name: 'copy-pinia',
-    async generateBundle() {
-      const filePath = path.resolve(
-        __dirname,
-        '../pinia/dist/pinia.esm-bundler.js'
-      )
+// function copyPiniaPlugin(): Plugin {
```

Comment:
this copied file does not seem to be used anywhere...?

Regardless it was not working, and it appears the actual file is `../pinia/dist/pinia.esm-browser.js`